strikecircle/env.py

```diff
@@ -0,0 +1,8 @@
+import environ
```

Comment:
Since this is identical to `sunrise/settings/env.py`, I think it might make sense to move this up into the base project directory, and then change both `strikecircle/tasks.py` and `sunrise/settings/common.py` to reference that single `env.py` file.